lib/OrePAN2/Index.pm

```diff
@@ -1,22 +1,17 @@
 package OrePAN2::Index;
 
-use strict;
-use warnings;
```

Comment:
If we're going to remove `strict` and `warnings`, could we move `use Moo` to line 3? That way there's no logic before `strict` and `warnings` are enabled.